xcat3/common/rest.py

```diff
@@ -0,0 +1,45 @@
+from oslo_log import log as logging
+from xcat3.common import exception
+
+import requests
+import json
+
+LOG = logging.getLogger(__name__)
+
+headers = {'Content-Type': 'application/json'}
+
+def login (url, in_data) :
+    session = requests.Session()
+    data = json.dumps(in_data)
+    print data
+    response = session.post(url, data=data, headers=headers, verify = False)
+
+    print response.text
+
+    if response.status_code != 200 :
```

Comment:
If it is a common library, what is the correct return code of http request? Does 201 or 202 works?